ChangeLog

```diff
@@ -48,6 +48,8 @@ Bugfix
    * Fix issue in RSA key generation program programs/x509/rsa_genkey
      where the failure of CTR DRBG initialization lead to freeing an
      RSA context without proper initialization beforehand.
+   * Fix bug in get_one_and_zeros_padding that sometimes accepted invalid
+     padding. Found and fixed by Micha Kraus.
```

Comment:
It would be good to be more specific here; in particular, it should be noted that this padding is not used in SSL/TLS as far as I see.

Reply:
For example, we might name the relevant padding identifier `{MBEDTLS/POLARSSL}_PADDING_ONE_AND_ZEROS` to be used in `[mbedtls_]cipher_set_padding_mode`.

Reply:
How about this?

> Fix bug in cipher decryption with `MBEDTLS_PADDING_ONE_AND_ZEROS` that sometimes accepted invalid padding. (Not used in TLS.) Found and fixed by Micha Kraus.

Reply:
@gilles-peskine-arm Sounds good to me, thanks!

Reply:
@hanno-arm @Patater I forced-push a replacement for the last commit in the PR, with the changelog entry tweaked per the discussion here.